SmartPlaces.Facilities/lib/OntologyMapper/src/MappingFileException.cs

```diff
@@ -19,6 +19,12 @@ public MappingFileException(string message, string filename)
             this.filename = filename;
         }
 
+        public MappingFileException(string message, string filename, Exception innerException)
+            : base(message, innerException)
+        {
+            this.filename = filename;
+        }
```

Comment:
From the parent PR, would like to see this move into an Exceptions folder. It can keep the `Microsoft.SmartPlaces.Facilities.OntologyMapper` namespace

Reply:
Fixed in most recent commit.